packages/expo-ui/ios/SwitchView.swift

```diff
@@ -20,7 +21,13 @@ struct SwitchView: ExpoSwiftUI.View {
   }
 
   var body: some View {
-    Toggle(isOn: $checked, label: { props.label != nil ? Text(props.label ?? "") : nil })
+    Toggle(isOn: $checked, label: {
+      if let label = props.label, let systemImage = props.systemImage {
+        Label(label, systemImage: systemImage)
+      } else if let label = props.label {
+        Text(label)
+      }
+    })
```

Comment:
This is somewhat unrelated to this PR but since we're changing this code can we replace the existing Toggle constructor with below API from [Toggle](https://developer.apple.com/documentation/swiftui/toggle#overview)? It accepts systemImage as well as label text. Because currently there is no way to set `foregroundColor` on label color. 

```
Toggle(
    "Vibrate on Ring",
    systemImage: "dot.radiowaves.left.and.right",
    isOn: $vibrateOnRing
)
```

In future we can introduce children API to `Switch` that can accept `Label` component.